addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5330,7 +5339,7 @@ msgstr ""
 
 #: unknown
 msgctxt "#13109"
-msgid "Test & apply resolution"
+msgid "Test & apply the resolution"
```

Comment:
I only wanted to say that the change is fine depending on where/how this string is used, and this needs to be checked.